projects/ion/src/lib/triple-toggle/triple-toggle.component.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { fireEvent, render, screen } from '@testing-library/angular';
+import { TripleToggleProps } from '../core/types';
+import { IonSharedModule } from '../shared.module';
+import { IonTooltipModule } from '../tooltip/tooltip.module';
+import { SafeAny } from '../utils/safe-any';
+import { IonTripleToggleComponent } from './triple-toggle.component';
+
+const tripleToggleId = 'ion-triple-toggle';
+const firstOptionId = 'btn-Sim';
+const middleOptionId = 'btn--';
+const lastOptionId = 'btn-Não';
+const selectedOption = 'ion-btn-primary';
+const notSelectedOption = 'ion-btn-secondary';
+
+const sut = async (customProps: TripleToggleProps = {}): Promise<void> => {
+  await render(IonTripleToggleComponent, {
+    componentProperties: customProps,
+    imports: [IonSharedModule, IonTooltipModule],
+  });
+};
+
+describe('IonTripleToggleComponent', () => {
+  describe('component basics', () => {
+    const checkEvent = jest.fn();
+
+    beforeEach(async () => {
+      await sut({
+        ionClick: {
+          emit: checkEvent,
+        } as SafeAny,
+      });
+    });
+    it('should render triple toggle', async () => {
+      expect(screen.getByTestId(tripleToggleId)).toBeInTheDocument();
+    });
+    it('should render middle option selected', async () => {
+      expect(screen.getByTestId(middleOptionId)).toHaveClass(selectedOption);
+    });
+    it('should render first and last option not selected', async () => {
+      expect(screen.getByTestId(firstOptionId)).not.toHaveClass(
+        'ion-btn-primary'
+      );
+      expect(screen.getByTestId(lastOptionId)).not.toHaveClass(
+        'ion-btn-primary'
+      );
+    });
+    it('should select when clicked', async () => {
+      const element = screen.getByTestId(firstOptionId);
+      fireEvent.click(element);
```

Comment:
```suggestion
      const firstOption = screen.getByTestId(firstOptionId);
      fireEvent.click(firstOption);
```
I think its better to specify the element you are clicking with the const name.